contrib/peer-logging/peer-logging-parser.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Parse message log binary files.  To be used in conjunction with -logmessages."""
+
+import argparse
+import os
+import sys
+from io import BytesIO
+import json
+from pathlib import Path
+from typing import Any, List
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '../../test/functional'))
+
+from test_framework.messages import ser_uint256     # noqa: E402
+from test_framework.mininode import MESSAGEMAP      # noqa: E402
+
+TIME_SIZE = 8
+LENGTH_SIZE = 4
+COMMAND_SIZE = 12
```

Comment:
The "command" terminology should not be used anymore for network messages, we just call it "message type" now (see e.g. PRs https://github.com/bitcoin/bitcoin/pull/18533, https://github.com/bitcoin/bitcoin/pull/18610, https://github.com/bitcoin/bitcoin/pull/18937). Replacement suggestions: . s/COMMAND_SIZE/MSGTYPE_SIZE and s/command/msg_type (in the parsing code below).

Reply:
Thanks for catching this!  This "command -> message type" switch is a monumental effort that I'm happy to be a part of.

I'm going to choose `msgtype` not `msg_type` for the same reasons you did in #18610 :)